app/Component/ViewActions.js

```diff
@@ -38,7 +47,7 @@ class ViewActions extends React.Component {
 
                     return <MaDeleteButton key={i++} entityName={entityName} entry={entry} size={size} />;
                 default:
-                    return React.createElement(button);
+                    return eval(jsx.fromString(button, templateParams));
```

Comment:
why don't you use the compile component here? It's not executed in the right context?